### PR TITLE
added init which returns definitly count of domains

### PR DESCRIPTION
**Pull request by OlegPhenomenon**
Close #[https://github.com/internetee/registrant_center/issues/54](url)

### CODE REVIEW

**Comment by OlegPhenomenon** [comment]
Related to -> https://github.com/internetee/registrant_center/issues/54

File: app/controllers/api/v1/registrant/domains_controller.rb [hunk 1]
```diff
@@ -57,10 +59,34 @@ def current_user_domains_total_count
         end
 
         def current_user_domains
-          current_registrant_user.domains(admin: params[:tech] != 'true')
+          initialization_count_of__domains
         rescue CompanyRegister::NotAvailableError
+          initialization_count_of__direct_domains
+        end
+
+        def initialization_count_of__direct_domains
```
Comment (yulgolem) [inline app/controllers/api/v1/registrant/domains_controller.rb:67]: Double underscore?

File: app/controllers/api/v1/registrant/domains_controller.rb [hunk 2]
```diff
@@ -57,10 +59,34 @@ def current_user_domains_total_count
...
+              return current_registrant_user.direct_domains(admin: false)
+            end
+
+            current_registrant_user.direct_domains(admin: true)
+          end
+
           current_registrant_user.direct_domains(admin: params[:tech] != 'true')
         end
+
+        def initialization_count_of__domains
```
Comment (yulgolem) [inline app/controllers/api/v1/registrant/domains_controller.rb:79]: Here as well

**Review by yulgolem — CHANGES_REQUESTED** [review]
In Rails way "initializer" is a config file for some external lib/gem/service put into `/config/initializers` folder, so I'd ask to consider renaming the PR and entities.

BTW why shouldn't we call method just something like 'init_count_*'?

**Review by yulgolem — APPROVED** [review]

**Pull request merged**